src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -470,6 +483,16 @@ protected function save_widget( $request ) {
 			$form_data = array();
 		}
 
+		/**
+		 * Fires after a widget is created or updated via the REST API.
+		 *
+		 * @since 5.8.0
+		 * @param WP_Widget       $widget_object Inserted or updated widget object.
```

Comment:
`$widget_object` isn't actually very useful for third parties as it doesn't have any of the widget's data. (It contains logic about how to render and save a widget.) Better to send `$widget_id` and `$sidebar_id`, same as you did in `rest_delete_widget`.